app/docker/views/images/build/buildimage.html

```diff
@@ -6,7 +6,7 @@
       <rd-widget-body>
         <uib-tabset active="state.activeTab">
           <uib-tab index="0">
-            <uib-tab-heading class="vertical-center"> <pr-icon icon="'tool'" class="leading-none"></pr-icon> Builder </uib-tab-heading>
+            <uib-tab-heading class="vertical-center"> <pr-icon icon="'wrench'" class="leading-none"></pr-icon> Builder </uib-tab-heading>
```

Comment:
Not sure if intentional, but this change is not present in the EE pr.

Reply:
Nice spot - Will add this to the EE pr - the wrench should be used